Client/Frontend/Overlay/Content/Menu/TrackingProtection/TrackingMenuSettingsView.swift

```diff
@@ -78,22 +51,34 @@ struct TrackingSettingsBlock: View {
     @Default(.contentBlockingStrength) private var contentBlockingStrength
 
     var body: some View {
-        // Toggle("Block tracking cookies", isOn: $blockTrackingCookies)
-        // Toggle("Block tracking requests", isOn: $blockTrackingRequests)
-        // Toggle("Update requests to HTTPS", isOn: $upgradeToHTTPS)
-        Toggle("Enable Protection", isOn: $contentBlockingEnabled)
+        Section(
+            header: Text("TRACKING PROTECTION")
+        ) {
+            Toggle("Enable Protection", isOn: $contentBlockingEnabled)
+        }
 
-        Picker("Protection Mode", selection: $contentBlockingStrength) {
-            ForEach(BlockingStrength.allCases) { strength in
-                Text(strength.name.capitalized)
+        Section(
+            footer: Text(
+                "If a site doesn't work as expected, you can deactivate Cookie Cutter for the site at any time."
+            )
+        ) {
+            Picker("Protection Mode", selection: $contentBlockingStrength) {
+                ForEach(BlockingStrength.allCases) { strength in
+                    VStack(alignment: .leading) {
+                        Text(strength.name.capitalized)
+                            .padding(.bottom, -4)
+                        Text(strength.description).withFont(.bodySmall).foregroundColor(
+                            .secondaryLabel
+                        ).frame(width: geometrySize.width - 120, alignment: .leading)
+                    }
                     .tag(strength.rawValue)
+                }
+                .padding(.top, 5)
```

Comment:
Can make this one line:

```suggestion
                .padding(.vertical, 5)
```